website/server/libs/user/index.js

```diff
@@ -102,6 +102,17 @@ function checkPreferencePurchase (user, path, item) {
   return _.get(user.purchased, itemPath);
 }
 
+async function checkNewInputForProfanity (user, res, newValue) {
+  if (stringContainsProfanity(newValue, 'slur') || stringContainsProfanity(newValue)) {
+    if (stringContainsProfanity(newValue, 'slur')) {
+      user.flags.chatRevoked = true;
+      await user.save();
+      throw new BadRequest(res.t('bannedSlurUsedInProfile'));
+    }
+    throw new BadRequest(res.t('displaynameIssueSlur'));
```

Comment:
the string here should be changed maybe since it's not just about the display name anymore?

Reply:
I didn't change this since it felt like a pretty major change. 

It's now about the display name and the 'about' section, both of which pertain the same screen (user profile modal). I suppose 'userProfileIssueSlur' would have been slightly more accurate but, per the docs, I felt like it was close enough in context not to warrant changing the json keys in all the translation files.

Would you like me to change anyway?

Reply:
Also, do I need to do anything to alert anyone to new/edited translations required?

Reply:
Regarding the new / edited translations no other work is needed! They're handled automatically. I'll let @Alys decide on the string change